mappings/net/minecraft/entity/Bird.mapping

```diff
@@ -0,0 +1 @@
+CLASS app net/minecraft/entity/Bird
```

Comment:
BirdEntity?

Reply:
We also have `CrossbowUser` (rather than `CrossbowUserEntity`) and `Npc` (rather than `NpcEntity`) interfaces. It seems like the current convention is to not add the entity suffix to interfaces (technically, they could be implemented by non-entities too).

Reply:
Ah, I assumed this was a base-class rather than an interface.